index.html

```diff
@@ -8,6 +8,14 @@
     <link rel="stylesheet" href="./styles.css" />
   </head>
   <body>
+    <nav>
+      <a href="https://forum.freecodecamp.org/" target="_blank">
+        <img
+          src="https://design-style-guide.freecodecamp.org/downloads/fcc_primary_large.png;"
```

Comment:
It would be better to have this image right  in the project instead of relying on the external url because if freecodecamp decides to change that url then our image would break.